src/main/java/th/ac/kmitl/soa/group2/utils/Time.java

```diff
@@ -0,0 +1,11 @@
+package th.ac.kmitl.soa.group2.utils;
+
+import java.sql.Timestamp;
+
+public class Time {
+
+    public static Timestamp now() {
+        return new Timestamp(System.currentTimeMillis());
```

Comment:
I see this one use only once, Why we over designed here ?
Also this is makes some sense when it has complex logic or time zone relate but this seems to be just simple timestamp.

Reply:
For readability sake. I think `Time.now()` is more expressive than `new Timestamp(System.currentTimeMillis())`